ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -22,12 +22,15 @@ elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
   master_type=Standard_D8s_v3
 fi
 
-cat >> "${CONFIG}" << EOF
+if [ X"${OUTBOUND_TYPE}" == X"UserDefinedRouting" ]; then
```

Comment:
The better approach should be something like: https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L74-L86

Create a patch yaml file for your setting, then patch it to the existing install-config.yaml.